frontend/src/APIClients/lessonAPIClient.ts

```diff
@@ -0,0 +1,89 @@
+import baseAPIClient from "./BaseAPIClient";
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+export type LessonRequest = {
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Record<string, any>];
+};
+
+export type LessonResponse = {
+  id: string | number;
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Record<string, any>];
+};
+
+// TO DO: error handling
+
+const getLesson = async (id: string): Promise<LessonResponse> => {
+  const bearerToken = `Bearer ${getLocalStorageObjProperty(
+    AUTHENTICATED_USER_KEY,
+    "accessToken",
+  )}`;
+  try {
+    const { data } = await baseAPIClient.get(`/lesson${id}`, {
```

Comment:
What's the difference between `/lesson/${id}` and `/lesson${id}`? Starter code seems to follow the former

Reply:
Good catch! It should be `/lesson/${id}`